filename.go

```diff
@@ -0,0 +1,10 @@
+package slinguist
+
+func GetLanguageByFilename(filename string) (lang string, safe bool) {
```

Comment:
safe is used as a part of the process to fall through the different strategies to detect the language used in a source file. Maybe safe would be described in other place of the code (GetLanguage function, README?) and in other PR.